app/src/main/res/drawable-v21/clickable_purple.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+        android:color="@color/black_alpha_12"
+        >
+
+    <item
+            android:id="@android:id/mask"
+            android:drawable="@android:color/white"
+            />
+
+    <item android:drawable="@color/purple_alpha_15" />
```

Comment:
I specified this color, but maybe the different color is better.
It's another issue, so no need to change it on this PR! 👍